src/Groups/types.jl

```diff
@@ -302,10 +395,12 @@ function _oscar_group(obj::GapObj, G::MatrixGroup)
   return M
 end
 
+
 ################################################################################
 #
 # "Coerce" an Oscar group `G` to one that is compatible with
 # the given Oscar group `S`.
+#T what does compatible mean?
 compatible_group(G::T, S::T) where T <: GAPGroup = _oscar_group(GapObj(G), S)
```

Comment:
This is not used anywhere, is it? Can we just remove it?

Reply:
Yes.